chore: remove beta info (#937)
netlify/netlify-plugin-nextjs: README.md

Version 4 is a complete rewrite of the Essential Next.js plugin. For full details of everything that's new, check out
[the v4 release notes](https://github.com/netlify/netlify-plugin-nextjs/blob/main/docs/release-notes/v4.md)

## Installing the plugin

> The plugin should be installed automatically for new Next.js sites on Netlify. You can also install it manually like

Reviewer: [sand] Minor suggestion for concision.
```suggestion
The plugin installs automatically for new Next.js sites on Netlify. You can also install it manually like
```

this:

- Install the module:

```shell
npm install -D @netlify/plugin-nextjs
```

...then add it to your `netlify.toml` file:

```toml
[[plugins]]
package = "@netlify/plugin-nextjs"
```

## Migrating from an older version of the plugin

You can manually upgrade from the previous version of the plugin by running the following command:

```shell
npm install -D @netlify/plugin-nextjs@latest
```

Change the `publish` directory to `.next`:

```toml
[build]
publish = ".next"
```

> If you previously set a custom `distDir` in your `next.config.js`, or set `node_bundler = "esbuild"` or
> `external_node_modules` in your `netlify.toml` these are no longer needed and can be removed.

Reviewer: [sand] Structure edit to improve readability.

```suggestion
If you previously set these values, they're no longer needed and can be removed:
- `distDir` in your `next.config.js`
- `node_bundler = "esbuild"` in `netlify.toml`
- `external_node_modules` in `netlify.toml`
```
